Plugins/JiraCommitHintPlugin/JiraCommitHintPlugin.cs

```diff
@@ -253,13 +254,36 @@ private void gitUiCommands_PostRepositoryChanged(object sender, GitUIEventArgs e
             _currentMessages = null;
         }
 
+        private static IDictionary<string, object> GetFlatIssueTemplateDictionary(Issue issue)
```

Comment:
Speculating, but if you pass `issue.CustomFields` instead of `issue` into the method it should be simpler to test.
Please add tests